routes/web.php

```diff
@@ -102,6 +102,10 @@
                 'as' => 'users.resets.create',
                 'uses' => 'Admin\UserController@sendPasswordReset',
             ]);
+            Route::post('users/{user}/mute-promotions', [
+                'as' => 'users.mute-promotions',
+                'uses' => 'Admin\UserController@mutePromotions',
+            ]);
```

Comment:
Also subjective, but I like the idea of keeping this RESTful & considering this a `promotions` sub-resource on the user that we can delete (and following the same convention for the route alias):

```suggestion
            Route::delete('users/{user}/promotions', [
                'as' => 'users.promotions.destroy',
                'uses' => 'Admin\PromotionsController@destroy',
            ]);
```